### ISSUE TITLE
Crash on build using appsee 2.1.4

### ISSUE THREAD

**Issue by AlbertoRoomi** [rollout/rollout.io-ios#60]
I Using Rollout.io (0.10.1) added via cocoapods when i try to update appsee from 2.1.3 to 2.1.4 to add iOS 9 compatibility I get this error and the build fails.

`Rollout post compile script
Rollout: going to generate structure and code
Rollout: using dsym generated by Xcode: "/.../Xcode/DerivedData/(Appname)-cynrxlevrkbeoubshkgiktydbpjd/Build/Products/Debug-iphoneos/Roomi (staging).app.dSYM/Contents/Resources/DWARF/Roomi (staging)"
http://error.rollout.io
/Library/Ruby/Gems/2.0.0/gems/json-1.8.3/lib/json/common.rb:155:in 'parse': 399: unexpected token at '{"arch":"armv7","data":}] (JSON::ParserError)
,"md5":"1c4b1b79be7b44d5502769d3702ee5e2","env_GCC_PREPROCESSOR_DEFINITIONS__DEBUG":"1","rollout_appKey":"x","CFBundleShortVersionString":"2.2.1.S","CFBundleVersion":"14","rollout_build":"128","structure_version":"1"}'
	from /Library/Ruby/Gems/2.0.0/gems/json-1.8.3/lib/json/common.rb:155:in 'parse'
	from /.../Pods/Rollout.io/lib/create_dynamic_swizzler.rb:84:in '<main>'
Command /bin/sh failed with exit code 3`
 

When i get back to appsee 2.1.3 it works, but I need to update it.

**Comment by silyevsk** [comment]
TLDR: The issue is fixed in Rollout SDK 0.10.2 .


It turns out that Xcode 7 saves debug symbols for compiled units (the .o files) in a new (optimized) way, making it harder to analyze them. Rollout SDK analyzes the final dsym, which can be still analyzed with our current tools if the end project is built with Xcode 7. However, if there's a library compiled with Xcode 7 (like Appsee in this case), and the final project is built with Xcode 6, then the dsym can't be analyzed with our current tools.

Appsee 2.1.4 had an additional problem with its debug symbols due to some Xcode 7 bug and so it was not possible to use it together with Rollout even with Xcode 7. AFAIK Appsee has now rebuilt 2.1.4 without debug symbols , so it should not be the issue anymore.

There were several crashes/issues in the Rollout.io post-compile build phase due to the above reasons which we've fixed in 0.10.2, so even if you use Xcode 6 and the older Appsee, you shouldn't see the errors in your build. You might however see internal Rollout warnings (since it's unable to fully analyze the dsym), but those are shown only in the build phase log and are not build warnings.
